stdlib/public/core/IntegerParsing.swift

```diff
@@ -83,6 +83,16 @@ where CodeUnits.Element : UnsignedInteger {
   }
 }
 
+@inline(never)
```

Comment:
Why isn't this an extension method on `FixedWidthInteger`?  As an implementation detail of that `init?` it rather makes sense to put it right next to it in the type.